tests/model/model.cpp

```diff
@@ -552,3 +554,57 @@ TEST(Model, setAndCheckIdsAllEntities)
     const std::string a = printer->printModel(m);
     EXPECT_EQ(a, e);
 }
+
+TEST(Model, missingUnitsFromImportOfCnTerms)
```

Comment:
For me half of this test should be deleted, we only want/need to see the case where the bug was found i.e. the import case.

Reply:
It was to show the difference between the parsing and the importing as their behaviour should be consistent.  I guess now it's fixed the parse half can go.

Reply:
(oops, replied to wrong thing).